chef_master/source/install_supermarket.rst

```diff
@@ -291,6 +291,57 @@ When the bootstrap operation is finished, do the following:
 
       $ sudo chef-client
 
+Install Supermarket Directly (without a cookbook)
+=====================================================
+While there are many benefits to using the cookbook method to install Supermarket, there are also cases where it's simpler to set up the Supermarket installation manually. These steps will walk you through the process of manually configuring your private Supermarket server.
+
+Before following these steps, be sure to complete the OAuth setup process detailed in the `Chef Identity </install_supermarket.html#chef-identity>`__ section of this guide.  
+
+#. `Download <https://downloads.chef.io/supermarket/>`__ the correct package for your operating system from ``downloads.chef.io``. 
+
+#. Install Supermarket using the appropriate package manager for your distribution:
+
+   * For Ubuntu:
+
+     .. code-block:: bash
+
+        dpkg -i /path/to/package/supermarket*.deb
+
+   * For RHEL / CentOS:
+
+     .. code-block:: bash
+
+        rpm -Uvh /path/to/package/supermarket*.rpm
+
+#. Run the ``reconfigure`` command to complete the initial installation:
+
+   .. code-block:: none
+
+      sudo supermarket-ctl reconfigure
+
+#. Create an ``/etc/supermarket/supermarket.json`` file and add the following information, substituting the values for each configuration option with the OAuth 2.0 client credentials that were created in the `previous section </install_supermarket.html#chef-identity>`__:
+
+   .. code-block:: ruby
+
+      {
+          "chef_server_url": "https://chefserver.mycompany.com:443",
+          "chef_oauth2_app_id": "0bad0f2eb04e935718e081fb71asdfec3681c81acb9968a8e1e32451d08b",
+          "chef_oauth2_secret": "17cf1141cc971a10ce307611beda7ffadstr4f1bc98d9f9ca76b9b127879",
+          "chef_oauth2_verify_ssl": false
```

Comment:
This list leaves out `chef_identity_url` which is necessary to set in the latest stable version but that will quickly be fixed by what's coming in a release soon.

We should also add setting `fqdn`to these steps for when the private Supermarket is to serve itself up as a name that's different than the hostname of the server.